capstone-app/src/app/app.component.ts

```diff
@@ -1,15 +1,27 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ContentComponent } from './modules/onboarding/content/content.component';
+import { MatDialog } from '@angular/material/dialog';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'GMapr';
   activeMarker: string;
   city: string;
 
+  constructor(public dialog: MatDialog) { }
+
+  openDialog() {
+    this.dialog.open(ContentComponent);
+  }
```

Comment:
Is there a reason this one line function needs to be its own? Can't we place the line in ngOnInit?